src/Microsoft.Health.Fhir.CosmosDb.Initialization/Features/Storage/StoredProcedures/DataPlaneStoredProcedureInstaller.cs

```diff
@@ -9,24 +9,26 @@
 using System.Threading.Tasks;
 using EnsureThat;
 using Microsoft.Azure.Cosmos;
-using Microsoft.Health.Fhir.CosmosDb.Features.Storage.Versioning;
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage.StoredProcedures;
 
-namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage.StoredProcedures
+namespace Microsoft.Health.Fhir.CosmosDb.Initialization.Features.Storage.StoredProcedures
 {
-    public class StoredProcedureInstaller : ICollectionUpdater
+    public class DataPlaneStoredProcedureInstaller : IStoredProcedureInstaller
     {
-        private readonly IEnumerable<IStoredProcedure> _storedProcedures;
+        private readonly IEnumerable<IStoredProcedureMetadata> _storedProcedures;
```

Comment:
I think it makes sense to rename this private attribute to _storeProceduresMetadata.

Reply:
done